man1/aur-build.1

```diff
@@ -104,19 +107,19 @@ For example:
 
 .EE
 
-.SY Note:
-If the rule should only apply to certain hosts, replace \fBALL\fR with 
+.SY Note :
+If the rule should only apply to certain hosts, replace \fBALL\fR with
 the respective \fIhostname\fR.
 
 .SH SEE ALSO
-.BR aur (1),
-.BR aur\-chroot (1),
-.BR pacconf (1),
-.BR pacman (1),
-.BR makepkg.conf (5),
-.BR pacman.conf (5),
-.BR makepkg (8),
-.BR repo-add (8),
+.BR aur (1) ,
```

Comment:
This is incorrect, there should be no space between `(1)` and `,`.  This would lead to the `,` being rendered as bold which is not the intent.

Reply:
Afaicr, this behavior with having punctuation as separate arguments only works in the mdoc extension, not man.